system-tests/tests/import-@

```diff
@@ -5,9 +5,10 @@ source ${SYSTESTS_PRELUDE} && prepare
 
 set -x -e -o pipefail
 
-distro=${TEST_NAME#import-}
+breed=$(echo "$TEST_NAME" | cut -d- -f2)
+distro=$(echo "$TEST_NAME" | cut -d- -f3)
 
-for iso in ${SYSTESTS_ROOT}/listings/${distro}/*; do
+for iso in ${SYSTESTS_ROOT}/listings/${breed}/${distro}/*; do
```

Comment:
do we need breed here?

Reply:
Well, otherwise we don't find the correct folder. So I assume that we indeed need it.

Reply:
Then I guess existing listings shall be renamed to match the new naming convention.